config/sample/host/01-ftc.yaml

```diff
@@ -692,3 +692,37 @@ spec:
     fields:
       - metadata.creationTimestamp
       - status
+---
+apiVersion: core.kubeadmiral.io/v1alpha1
+kind: FederatedTypeConfig
+metadata:
+  name: pods
+spec:
+  federatedType:
+    group: types.kubeadmiral.io
+    kind: FederatedPod
+    pluralName: federatedpods
+    scope: Namespaced
+    version: v1alpha1
+  targetType:
+    kind: Pod
+    pluralName: pods
+    scope: Namespaced
+    version: v1
+  sourceType:
+    kind: Pod
+    pluralName: pods
+    scope: Namespaced
+    version: v1
+  statusType:
+    group: types.kubeadmiral.io
+    kind: FederatedPodStatus
+    pluralName: federatedpodstatuses
+    scope: Namespaced
+    version: v1alpha1
+  controllers:
+    - - kubeadmiral.io/global-scheduler
+    - - kubeadmiral.io/overridepolicy-controller
+  statusCollection:
+    fields:
+      - status
```

Comment:
Should we also collect spec.nodeName?

Reply:
What happens when pod is evicted in the member?